### DISCUSSION TITLE
Change default currency

### DISCUSSION THREAD

**Discussion by domenico-ruggiano** [twentyhq/twenty#6933] [Ideas]
Possibility to set a default currency other than USD dollars, for example € EUR.

**Comment by FelixMalfait** [comment]
This should already be possible at the field level, if you go into your field's setting in the data model section in settings :)

**Comment by domenico-ruggiano** [comment]
@FelixMalfait  it's not possible to change the default currency for the Amount field inside the Opportunity data model:

https://github.com/user-attachments/assets/1e8bc662-fc2f-4af9-b85e-3a690767063c

**Reply by FelixMalfait** [reply]
Ah yes that's an issue indeed. You could always disable the field and re-create a new one on the same object in the meantime but we need to fix this

**Comment by FelixMalfait** [comment]
Closing as I created an issue to followup on this #7104
